app/components/baseComponents/components/table/table.tsx

```diff
@@ -1,28 +1,30 @@
 import { TableData }  from "../../models/tableData";
 import styles from "./table.module.scss";
 
-export default function Table({tableData, tableClassName}:{ tableData: TableData, tableClassName?:String }){
+
+const getTableRow = (tableRow:any={})=> {
+    return <tr className={styles.tableRow}>
+        <th className={styles.tableCell}>{tableRow.rank}</th>
+        <td className={styles.tableCell}>{tableRow.name}</td>
+        <td className={styles.tableCell}>{tableRow.totalScore}</td>
+    </tr>;
+};
```

Comment:
Shouldn't this work go in it's own PR? Or not... just took me a while to understand this diff because I wasn't expecting this here! 🙃

Reply:
Ahhh sure it could be. But this was under "Cleanup" since you wanted me to move this function from outside the Table function below.